runtime/parachains/src/runtime_api_impl/v1.rs

```diff
@@ -300,6 +300,8 @@ where
 				=> CandidateEvent::CandidateIncluded(c, h, core, group),
 			RawEvent::<T>::CandidateTimedOut(c, h, core)
 				=> CandidateEvent::CandidateTimedOut(c, h, core),
+			RawEvent::<T>::__Ignore(_, _)
+				=> unreachable!("__Ignore cannot be used"),
```

Comment:
Maybe change `frame_support` to not expose `__Ignore(_,_)` in Event api?

Reply:
i think unavoidable unfortunately, hence the name and stuff

Reply:
It is to capture the generics in case none of the fields capture it.

But it was more to make it handy we could not generate it like decl_event do.
Also right now there is a weird case that construct_runtime doesn't handle non generic or generic on `T` only for event when the pallet is instantiable. so in this case we sometime need an ignore variant.